kairon/api/app/routers/bot/bot.py

```
@@ -1705,6 +1706,27 @@ async def get_llm_logs(
     return Response(data=data)
 
 
+@router.get("/executor/logs", response_model=Response)
+async def get_executor_logs(
+        start_idx: int = 0, page_size: int = 10,
+        event_class: str = None, task_type: str = None,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS)
```

Ruff

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)